### ISSUE TITLE
Mask ProstT5 extra characters

### ISSUE THREAD

**Issue by linsalrob** [linsalrob/genome_entropy#55]
ProstT5 adds extra characters at the end of each sequence. We need to mask those after we get the outputs but before we get the residue embedding. Something like this block of code, but only for the prostt5 models

```
                        # ProstT5 appends a special tokens at the end of each sequence
                        # Mask this also out during inference while taking into account the prostt5 prefix

                        for idx, seq in enumerate(seq):
                            token_encoding.attention_mask[idx, len(seq) + 1] = 0
```
